aws/resource_aws_config_remediation_configuration_test.go

```diff
@@ -0,0 +1,324 @@
+package aws
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/configservice"
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+func testAccConfigRemediationConfiguration_basic(t *testing.T) {
+	var rc configservice.RemediationConfiguration
+	rInt := acctest.RandInt()
+	expectedName := fmt.Sprintf("tf-acc-test-%d", rInt)
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckConfigRemediationConfigurationDestroy,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccConfigRemediationConfigurationConfig_basic(rInt),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckConfigRemediationConfigurationExists("aws_config_remediation_configuration.foo", &rc),
+					testAccCheckConfigRemediationConfigurationName("aws_config_remediation_configuration.foo", expectedName, &rc),
+					resource.TestCheckResourceAttr("aws_config_remediation_configuration.foo", "config_rule_name", expectedName),
+					resource.TestCheckResourceAttr("aws_config_remediation_configuration.foo", "target_id", "SSM_DOCUMENT"),
+					resource.TestCheckResourceAttr("aws_config_remediation_configuration.foo", "target_type", "AWS-PublishSNSNotification"),
+					resource.TestCheckResourceAttr("aws_config_remediation_configuration.foo", "parameters.0.resource_value", "Message"),
+				),
+			},
+		},
+	})
+}
+
+func testAccConfigRemediationConfiguration_importAws(t *testing.T) {
```

Comment:
Our [current recommendations for acceptance testing import](https://www.terraform.io/docs/extend/resources/import.html#recommendations-for-import) is to include the `ImportStateVerify: true` testing in every existing acceptance test instead of creating a separate import test. 👍